src/util/strb.h

```diff
@@ -77,6 +77,15 @@ static inline int strb_error(strb *sb) {
   return sb->l == (size_t)-1;
 }
 
+/*
+ * Initialize at runtime an strb.
+ */
+
+static inline void strb_init(strb* sb){
+  const strb s = STRB_STATIC_INIT;
+  *sb = s;
```

Comment:
@abergeron I think it would be wise to keep it as is, Don't Repeat Yourself and all that. If `STRB_STATIC_INIT` changes in such a way that `memset()` no longer suffices, this function will continue working. Otherwise there's two different places that must be changed, and one can forget one or the other.

At any rate, for the present `strb`, the compiler can and will optimize `memset()` to just
```
xor eax, eax
movq [sb   ], rax
movq [sb+ 8], rax
movq [sb+16], rax
```